src/components/functional/ShowMore.vue

```diff
@@ -49,7 +45,7 @@ export default {
   watch: {
     trace() {
       if (this.trace || this.prefix) {
-        this.expanded = this.prefix === this.trace || this.trace.startsWith(`${this.prefix}-`) || this.expanded;
+        this.expanded = this.trace.startsWith(`${this.prefix}-`) || this.expanded;
```

Comment:
Ah, it moved to here! Note that we use `ShowMore` in all places where we show/hide something (I might rename to `ShowHide` or `Toggle` at some point).

Reply:
Jupp, I'm aware. When I played around with this, everything started to pop out.